m2cgen/ast.py

```diff
@@ -48,18 +48,53 @@ def __str__(self):
         return "BinNumExpr(" + args + ")"
 
 
-class VectorExpr(NumExpr):
+# Vector Expressions.
+
+class VectorExpr(Expr):
     is_vector_output = True
 
+
+class VectorVal(VectorExpr):
```

Comment:
Can we call it `VectorValExpr` to maintain consistency with other AST members.

Reply:
Ack